docs/changelog/1.0.2.md

```diff
@@ -0,0 +1 @@
+[Feature]: implement localised content from Vendure api. [#142] https://github.com/vuestorefront-community/vendure/issues/142
```

Comment:
One more thing, shouldn't it be added to a changelog for the 1.0.1 version?

Reply:
I will clean up the changelogs before releasing so no worries :)